packages/app-store/zapier/components/TemplateCard.tsx

```diff
@@ -17,11 +17,11 @@ export default function TemplateCard({ template }: { template: Template }) {
             <p className="mt-[2px] text-xs text-gray-500 ">{template.text}</p>
```

Comment:
template.app should have `leading-4` and template text should be `text-sm `

Reply:
cc @om-ray

Reply:
I'll get this in right now!